rsearch/bin/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	search "github.com/romana/contrib/rsearch"
```

Comment:
Similar for this, should potentially be `k8s.io/contrib`.
It means you need to clone this repo manually to the location it would live in GOPATH.
Not really much choice for an upstream submission.